docs/source/basis/members.rst

```diff
@@ -69,7 +69,7 @@ To summarize:
     a [label="A value was previously set?"];
     a->b[label="No"];
     a->c[label="Yes"];
```

Comment:
And this "No"

Reply:
Later on the steps f and g have been inverted.

Reply:
Done